plugins/VstEffect/VstEffectControlDialog.cpp

```diff
@@ -272,7 +272,7 @@ void VstEffectControlDialog::showEvent(QShowEvent *_se)
 
 VstEffectControlDialog::~VstEffectControlDialog()
 {
-#if !(QT_VERSION < 0x050000 && defined(LMMS_BUILD_LINUX))
+#ifndef LMMS_BUILD_LINUX
```

Comment:
The condition is now always true(`!(false && something) == !false == true`). So this check should be dropped.

Reply:
Right! Thanks.